pkg/handlers/mount.go

```diff
@@ -120,10 +120,11 @@ func Mount(allowedFilesystems map[string]struct{}, requireUserNamespaceAdmin boo
 		}
 
 		var options string
-		if req.Data.Args[4] != 0/* NULL */ && filesystem != "proc" && filesystem != "sysfs" {
+		if req.Data.Args[4] != 0/* NULL */ && filesystem != "sysfs" {
 			// Get options, we assume because this is specified in
 			// allowedFilesystems that the data argument to mount(2)
-			// is a string so this is safe now.
+			// is a string so this is safe now. We ignore options for sysfs, as it
+			// doesn't define options.
```

Comment:
For sysfs this is not executed, right? We just changed the if and only enters if filesystem != sysfs

Reply:
Yes, the intention is it doesn't even read an option if it's sysfs. It may well be overly paranoid as sysfs doesn't take any option per https://docs.kernel.org/filesystems/sysfs.html

Reply:
Oh, okay, I understand what you meant with the comment now. I misread, sorry :D